packages/back-end/src/models/GithubUserTokenModel.ts

```diff
@@ -0,0 +1,77 @@
+import { omit } from "lodash";
+import uniqid from "uniqid";
+import mongoose from "mongoose";
+import { refreshToken } from "@octokit/oauth-methods";
+import {
+  GithubUserTokenInterface,
+  CreateGithubUserTokenInput,
+} from "../../types/github";
+
+type GithubUserTokenDocument = mongoose.Document & GithubUserTokenInterface;
+
+const githubUserTokenSchema = new mongoose.Schema({
+  id: String,
```

Comment:
We should have an `organization` field for every data model and use it in all queries.  It's the safest way to prevent critical cross-account access bugs.